doc/languages-frameworks/gnome.xml

```diff
@@ -40,6 +40,23 @@
    </para>
   </section>
 
+  <section xml:id="ssec-icon-theme-packaging">
+    <title>Packaging icon themes</title>
+
```

Comment:
I believe this whole chapter is about packaging, so why not expand what is above?

Reply:
I think separate section is fine, since there is not much overlap between apps and themes.